@navikt/core/react/src/form/form-summary/FormSummaryHeading.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { forwardRef } from "react";
+import { Heading, HeadingProps } from "../../typography";
+import { OverridableComponent } from "../../util/types";
+
+export interface FormSummaryHeadingProps
+  extends React.HTMLAttributes<HTMLHeadingElement> {
+  /**
+   * The heading level.
+   *
+   * @default "3"
+   */
+  level?: HeadingProps["level"];
```

Comment:
Bør `level="1"` omittes her?

Reply:
så vanskelig det skulle være å bruke `Omit<>` her 🤔, bruker jeg `as HeadingProps["level"]` så kan man ungå TS feil, men det er jo skjeldent riktig + jeg får ikke feil når jeg tester med `level="1"` heller 🤔. På en måte hadde jeg håpet at TS skulle kunne se at ene typen er et direkte subset av den andre, og ikke slå seg vrang?!

![image](https://github.com/navikt/aksel/assets/819074/115d1a84-0506-4f83-8bb1-f80411676b6e)

Reply:
Pick er for objekter. Prøv Exclude.

Reply:
Det var det! 🙌